lib/utils/preferences.dart

```diff
@@ -24,6 +25,64 @@ Future<void> performSharedPreferencesMigration() async {
   bool? legacyCommentUseColorizedUsername = prefs.getBool(LocalSettings.commentUseColorizedUsername.name);
   if (legacyCommentUseColorizedUsername != null) {
     await prefs.remove(LocalSettings.commentUseColorizedUsername.name);
-    await prefs.setBool(LocalSettings.userFullNameColorizeUserName.name, legacyCommentUseColorizedUsername);
+    await prefs.setBool('user_fullname_colorize_user_name', legacyCommentUseColorizedUsername);
+  }
+
+  // Migrate legacy user/community styles

```

Comment:
Do we need to migrate these settings? I believe #1227 was never released in a nightly build so we might be able to get away with not having to do any migrations.

Reply:
Yeah good point! I was probably overzealous with the migrations, but realistically no one besides us has even had the chance to run this. 😆